x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/lucene/ValuesSourceReaderOperator.java

```diff
@@ -654,6 +654,16 @@ public BlockLoader.DoubleBuilder doubles(int expectedCount) {
             return factory.newDoubleBlockBuilder(expectedCount);
         }
 
+        @Override
+        public BlockLoader.FloatBuilder floatsFromDocValues(int expectedCount) {
+            return factory.newFloatBlockBuilder(expectedCount).mvOrdering(Block.MvOrdering.SORTED_ASCENDING);
+        }
+
+        @Override
+        public BlockLoader.FloatBuilder floats(int expectedCount) {
+            return factory.newFloatBlockBuilder(expectedCount);
```

Comment:
I added support for float blocks/vectors to the compute engine quite a while ago, happy to see them being used now 👍